backends/openvino/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Intel Corporation
+#
+# Licensed under the BSD License (the "License"); you may not use this file
+# except in compliance with the License. See the license file in the root
+# directory of this source tree for more details.
+
+# Set minimum required CMake version
+cmake_minimum_required(VERSION 3.19)
+
+# Set project name
+project(openvino_backend_project)
+
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Ensure compile_commands.json is generated
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# Set up EXECUTORCH_ROOT if not already set
+if(NOT EXECUTORCH_ROOT)
+  set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+endif()
+
+# Define common include directories
+set(COMMON_INCLUDE_DIRS ${EXECUTORCH_ROOT}/..)
+
+# Include utility CMake scripts from ExecuteTorch
+include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+
+# Set OpenVINO directory from environment variable
+set(OPENVINO_DIR "$ENV{INTEL_OPENVINO_DIR}")
```

Comment:
could you please just use:
```cmake
find_package(OpenVINO REQUIRED)
```
As I see this code with `$ENV{INTEL_OPENVINO_DIR}` is copied from ONNX RT integration and it's quite obsolete approach as OpenVINO has mature cmake interface.

And later instead of dealing with `OPENVINO_LIB`, `OPENVINO_INCLUDE_DIRS`, `OPENVINO_LIB_PATH` you can just link:
```cmake
target_link_libraries(openvino_backend PRIVATE openvino::runtime executorch_core)
```

Reply:
Thanks for the suggestion. Updated the files accordingly.